docs/topics/http/decorators.txt

```diff
@@ -97,3 +97,21 @@ caching based on specific request headers.
     into account when building its cache key.
 
     See :ref:`using vary headers <using-vary-headers>`.
+
+.. module:: django.views.decorators.cache
+
+Caching
+=======
+
+The decorators in :mod:`django.views.decorators.cache` control server and
+client-side caching.
+
+.. function:: never_cache(view_func)
+
+    This decorator adds headers to ensure the response won’t be cached by
```

Comment:
no need to use left quote - sphinx will make the correct replacement on the rendered docs